fs/nfsd/nfs4acl.c

```diff
@@ -323,12 +350,12 @@ get_nfs4_nfsv41xdr_acl(struct svc_rqst *rqstp, struct dentry *dentry,
 
 int
 nfsd4_get_nfs4_acl(struct svc_rqst *rqstp, struct dentry *dentry,
-		struct nfs4_acl **acl)
+		struct nfs4_acl **acl, enum nfs4_acl_type acl_type)
```

Comment:
Yes.  Furthermore, since I was moving some code that only had sporadic `#if CONFIG_TRUENAS` handling, I did not update it everywhere.  (I was mostly using CONFIG_TRUENAS as flags to where I was tweaking code.)